README.md

```diff
@@ -345,7 +345,7 @@ No modules.
        = optional(number)<br>        net_ipv4_ip_local_port_range_min   = optional(number)<br>        net_ipv4_ip_local_port_range_max   = optional(number)<br>        net_ipv4_neigh_default_gc_thresh1  = optional(number)<br>        net_ipv4_neigh_default_gc_thresh2  = optional(number)<br>        net_ipv4_neigh_default_gc_thresh3  = optional(number)<br>        net_ipv4_tcp_fin_timeout           = optional(number)<br>        net_ipv4_tcp_keepalive_intvl       = optional(number)<br>        net_ipv4_tcp_keepalive_probes      = optional(number)<br>        net_ipv4_tcp_keepalive_time        = optional(number)<br>        net_ipv4_tcp_max_syn_backlog       = optional(number)<br>        net_ipv4_tcp_max_tw_buckets        = optional(number)<br>        net_ipv4_tcp_tw_reuse              = optional(bool)<br>        net_netfilter_nf_conntrack_buckets = optional(number)<br>        net_netfilter_nf_conntrack_max     = optional(number)<br>        vm_max_map_count                   = optional(number)<br>        vm_swappiness                      = optional(number)<br>        vm_vfs_cache_pressure              = optional(number)<br>      }))<br>      transparent_huge_page_enabled = optional(string)<br>      transparent_huge_page_defrag  = optional(string)<br>      swap_file_size_mb             = optional(number)<br>    }))<br>    fips_enabled       = optional(bool)<br>    kubelet_disk_type  = optional(string)<br>    max_count          = optional(number)<br>    max_pods           = optional(number)<br>    message_of_the_day = optional(string)<br>    mode               = optional(string, "User")<br>    min_count          = optional(number)<br>    node_network_profile = optional(object({<br>      node_public_ip_tags = optional(map(string))<br>    }))<br>    node_labels                  = optional(map(string))<br>    node_public_ip_prefix_id     = optional(string)<br>    node_taints                  = optional(list(string))<br>    orchestrator_version         = optional(string)<br>    os_disk_size_gb              = optional(number)<br>    os_disk_type                 = optional(string, "Managed")<br>    os_sku                       = optional(string)<br>    os_type                      = optional(string, "Linux")<br>    pod_subnet_id                = optional(string)<br>    priority                     = optional(string, "Regular")<br>    proximity_placement_group_id = optional(string)<br>    spot_max_price               = optional(number)<br>    scale_down_mode              = optional(string, "Delete")<br>    snapshot_id                  = optional(string)<br>    ultra_ssd_enabled            = optional(bool)<br>    vnet_subnet_id               = optional(string)<br>    upgrade_settings = optional(object({<br>      max_surge = string<br>    }))<br>    windows_profile = optional(object({<br>      outbound_nat_enabled = optional(bool, true)<br>    }))<br>    workload_runtime = optional(string)<br>    zones            = optional(set(string))<br>  }))</pre> | `{}` | no |
```

Comment:
This line is autogenerated from the `variables.tf` code. Can you fix the type in `variables.tf` ?